service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderService.java

```diff
@@ -61,38 +44,26 @@ public CaseManagementOrder addAction(CaseManagementOrder order, OrderAction orde
         return data;
     }
 
-    public OrderAction removeDocumentFromOrderAction(OrderAction orderAction) {
-        return orderAction.toBuilder().document(null).build();
-    }
-
-    public boolean isHearingDateInFuture(CaseData caseData) {
-        LocalDateTime hearingDate = hearingBookingService
-            .getHearingBookingByUUID(caseData.getHearingDetails(), caseData.getCaseManagementOrder().getId())
-            .getStartDate();
+    public LocalDate getIssuedDate(CaseManagementOrder caseManagementOrder) {
+        if (caseManagementOrder == null || isEmpty(caseManagementOrder.getDateOfIssue())) {
+            return time.now().toLocalDate();
```

Comment:
Hmm, not sure if this is obvious... not from the method name. I would expect to be returned optional and have this logic where I need this, I think it is hidden currently, maybe it should be reflected in method name at least? 

I don't like surprises, default value is! Another option would be to pass default as param, would be super obvious. 

I know it was probably here, but green line :(.

Reply:
was on my radar to address, will address here no worries, should be on the case management order object